protex/system.py

```diff
@@ -103,7 +105,7 @@ def __init__(
         self.record_charge_state = []
         self.canonical_name = canonical_name
         self.system = system
-        self.record_charge_state.append(self.endstate_charge)
+        self.record_charge_state.append(self.endstate_charge)  # Not used anywhere?
```

Comment:
I think this code adds the first charge to the record_charge_state parameter. The next charge is added at the next simulation step.